### PR TITLE
Validation::rules() now uses reuses add()

### PR DESCRIPTION
**Pull request by SidRoberts**

### CODE REVIEW

**Comment by sergeyklay** [comment]
Thanks

File: phalcon/validation.zep [hunk 1]
```diff
@@ -247,25 +247,10 @@ class Validation extends Injectable implements ValidationInterface
 
 		for validator in validators {
 			if validator instanceof ValidatorInterface {
-				if typeof field == "array" {
-					// Uniqueness validator for combination of fields is handled differently
-					if validator instanceof CombinedFieldsValidator {
-						let this->_combinedFieldsValidators[] = [field, validator];
-					}
-					else {
-						for singleField in field {
```
Comment (Jurigag) [inline phalcon/validation.zep]: @sergeyklay remove from this method `var singleField` when you will be doing some commit.

Comment (sergeyklay) [inline phalcon/validation.zep]: @SidRoberts could you please fix this?

Comment (Jurigag) [inline phalcon/validation.zep]: I mean just not to have warning when compiling :D

**Pull request merged**